jekyll/_cci2/configuration-reference.md

```diff
@@ -607,6 +607,13 @@ step that needs to upload logs or code-coverage data somewhere.
 
 A value of `on_fail` means that the step will run only if one of the preceding steps has failed (returns a non-zero exit code). It is common to use `on_fail` if you want to store some diagnostic data to help debug test failures, or to run custom notifications about the failure, such as sending emails or triggering alerts in chatrooms.
 
+``` YAML
+- run:
+    name: Cleanup cloud
+    command: # Bunch of stuff here
+    when: always # always cleanup whether job passes or fails
+```
```

Comment:
Let's use a real example here, such as uploading code coverage results. This is something that you want to do regardless of whether the tests pass or fail.

```yaml
- run:
    name: Upload CodeCov.io Data
    command: bash <(curl -s https://codecov.io/bash) -F unittests
    when: always
```

Reply:
Done